### PR TITLE
Add get sub groups to Admin API SDK

### PR DESCRIPTION
**Pull request by aron-666**

### CODE REVIEW

File: src/Keycloak.AuthServices.Sdk/Admin/IKeycloakGroupClient.cs [hunk 1]
```diff
@@ -54,6 +91,9 @@ Task<HttpResponseMessage> GetGroupWithResponseAsync(
         CancellationToken cancellationToken = default
     );
 
+
```
Comment (NikiforovAll) [inline src/Keycloak.AuthServices.Sdk/Admin/IKeycloakGroupClient.cs:94]: nit: remove redundant spaces

File: src/Keycloak.AuthServices.Sdk/Admin/KeycloakClient.cs [hunk 2]
```diff
@@ -376,6 +376,54 @@ public async Task<HttpResponseMessage> GetGroupsWithResponseAsync(
...
+        {
+            queryBuilder.Add("max", parameters.Max?.ToString(CultureInfo.InvariantCulture)!);
+        }
+
+        if (parameters.Search is not null)
+        {
+            queryBuilder.Add("search", parameters.Search.ToString()!);
+        }
+
+        var url = path + queryBuilder.ToQueryString();
```
Comment (NikiforovAll) [inline src/Keycloak.AuthServices.Sdk/Admin/KeycloakClient.cs:420]: do we need to ensure trailing '/' at the end of the path? Does it work?

**Review by NikiforovAll — CHANGES_REQUESTED** [review]
Thank you for the PR, let's work on some small things before we can merge it

File: src/Keycloak.AuthServices.Sdk/Admin/KeycloakClient.cs (hunk 2 above)
Comment (aron-666) [reply src/Keycloak.AuthServices.Sdk/Admin/KeycloakClient.cs:420]: No, I have tested it and it works.

**Comment by NikiforovAll** [comment]
Thank you for the contribution

**Pull request merged**